modules/apps/layout/layout-type-controller/layout-type-controller-control-panel/src/main/java/com/liferay/layout/type/controller/control/panel/internal/layout/type/controller/ControlPanelLayoutTypeController.java

```diff
@@ -150,7 +135,15 @@ protected void setServletContext(ServletContext servletContext) {
 
 	private static final String _VIEW_PAGE = "/layout/view/control_panel.jsp";
 
+	@Reference
```

Comment:
@JanisZhang This is actually not related to servletContext, you should put these kind of changes to a separate commit.